File: src/main/java/dev/tricht/lunaris/util/ErrorUtil.java
package dev.tricht.lunaris.util;

import javax.swing.*;

public class ErrorUtil {
    public static void showErrorDialogAndExit(String message) {
        JOptionPane.showMessageDialog(null, message, "Lunaris", JOptionPane.ERROR_MESSAGE);
        System.exit(1);
    }
}

File: src/main/java/dev/tricht/lunaris/util/ErrorUtil.kt
package dev.tricht.lunaris.util

import javax.swing.*
import kotlin.system.exitProcess

object ErrorUtil {
    fun showErrorDialogAndExit(message: String) {
        JOptionPane.showMessageDialog(null, message, "Lunaris", JOptionPane.ERROR_MESSAGE)
        exitProcess(1)
    }
}
